Assets/Scripts/VisibleObject/VisibleObject.cs

```
@@ -45,6 +47,12 @@ void Start()
             _rigidbody = rb;
             _rigidbody.isKinematic = true;
         }
+        boundBox = GetComponent<BoundBox>();
+        if (boundBox != null)
```

We should probably just let Unity throw an error here if the boundBox is missing. Unless there is going to be a case where we dont want to have the boundBox?